README.md

```diff
@@ -14,13 +14,14 @@ Features
 * Modules for group management
 * Modules for host management
 * Modules for hostgroup management
+* Modules for sudocmd management
 * Modules for topology management
 * Modules for user management
 
 Supported FreeIPA Versions
 --------------------------
 
-FreeIPA versions 4.6 and up are supported by all roles. 
+FreeIPA versions 4.6 and up are supported by all roles.
```

Comment:
Please do not mix the commit with "unrelated" changes. Create a extra PR for this, please.

Reply:
Fixed